tests/unit/sqlalchemy/test_dialect.py

```diff
@@ -196,14 +202,14 @@ def setup(self):
                     source="trino-sqlalchemy",
                 ),
             ),
-        ],
+        ] if sqlalchemy_version() >= "1.4" else [],  # make_url is not supported in sqlalchemy 1.3
```

Comment:
do docs need updating as well that the `URL` static helper can only be used with 1.4.x?

Reply:
Actually I fixed it now by adding a slash after the port. So now the helper is working for 1.3 and 1.4.